Tests/TestingApps/PaywallsTester/PaywallsTester/Products.storekit

```diff
@@ -37,8 +47,8 @@
   ],
   "settings" : {
     "_failTransactionsEnabled" : false,
-    "_locale" : "es_ES",
-    "_storefront" : "ESP",
+    "_locale" : "en_US",
+    "_storefront" : "USA",
```

Comment:
Just checking but any reason to change the storefront/locale? I'm ok with the change though.

Reply:
this is intentional, should've mention it in the description. This was causing the paywalls tester to have a mix of Spanish (subscription name) and English and figured we didn't want that.